package.json

```diff
@@ -95,7 +99,6 @@
         "subPermissions": [
           "ui-authorization-roles.settings.view",
           "roles.item.delete",
-          "roles.users.item.delete",
```

Comment:
Are role assignments automatically deleted by cascade when a role is deleted? If so, fine. If not, then either 
1. this permission needs to stay so that we can issue the requests to delete them, and then delete the role
2. this permission goes, and we need to add logic in the application to hide the delete button when the user has `ui-authorization-roles.settings.delete`, AND does not have permission to manage role assignment, AND the role has one or more users assigned

Reply:
According to the JIRA ticket description [UIROLES-125](https://folio-org.atlassian.net/browse/UIROLES-125) for `roles.users.item.delete`: "NOTE:  the backend handles this... when deleting a role the UI shouldn't (as AFAICT doesn't) first remove role/user assignments associated with the role being deleted."